lib/bolt/outputter/human.rb

```diff
@@ -269,78 +270,104 @@ def print_tasks(tasks, modulepath)
 
       # @param [Hash] task A hash representing the task
       def print_task_info(task)
-        # Building lots of strings...
-        pretty_params = +""
-        task_info = +""
+        params = []
+
         usage = if Bolt::Util.powershell?
                   +"Invoke-BoltTask -Name #{task.name} -Targets <targets>"
                 else
                   +"bolt task run #{task.name} --targets <targets>"
                 end
 
-        task.parameters&.each do |k, v|
-          pretty_params << "- #{k}: #{v['type'] || 'Any'}\n"
-          pretty_params << "    Default: #{v['default'].inspect}\n" if v.key?('default')
-          pretty_params << "    #{v['description']}\n" if v['description']
-          usage << if v['type'].start_with?("Optional")
-                     " [#{k}=<value>]"
-                   else
-                     " #{k}=<value>"
-                   end
-        end
+        module_path = task.files.first['path'].chomp("/tasks/#{task.files.first['name']}")
+        module_path = 'built-in module' if module_path.start_with?(Bolt::Config::Modulepath::MODULES_PATH)
 
         if task.supports_noop
           usage << Bolt::Util.powershell? ? '[-Noop]' : '[--noop]'
```

Comment:
Sorry I know this isn't part of your changes, but I'm getting a stacktrace here (with Ruby 2.6.3) that I verified in IRB, where Ruby does not like this syntax:
```
/opt/puppetlabs/bolt/lib/ruby/gems/2.7.0/gems/bolt-3.1.0/lib/bolt/outputter/human.rb:285:in `print_task_info': no implicit conversion of false into String (TypeError)
```
It looks like saving that conditional output to a var and then append it, it works. This is also failing in current Bolt, definitely not related to your changes - it just seems worth fixing here.

Reply:
This is my task metadata:
```
{
  "description": "Task with noop",
  "supports_noop": true
}
```

Reply:
Nice catch! 🙌